src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -478,7 +478,7 @@ void func_80866B20(EnDoor* this, PlayState* play) {
         Animation_PlayOnceSetSpeed(&this->knobDoor.skelAnime, sAnimations[this->knobDoor.animIndex],
                                    (player->stateFlags1 & PLAYER_STATE1_8000000) ? 0.75f : 1.5f);
         if (this->unk_1A6 != 0) {
-            gSaveContext.save.saveInfo.inventory.dungeonKeys[gSaveContext.mapIndex]--;
+            DUNGEON_KEY_COUNT(gSaveContext.mapIndex) = DUNGEON_KEY_COUNT(gSaveContext.mapIndex) - 1;
```

Comment:
```suggestion
            DUNGEON_KEY_COUNT(gSaveContext.mapIndex)--;
```

Reply:
Personally I don't like doing `MACRO--`, it looks kinda ugly

Reply:
Oh yeah, this was the very first thing I changed back before I fell down the bracket rabbit hole. Worth noting `DUNGEON_KEY_COUNT(gSaveContext.mapIndex)--;` doesn't match and wasn't sure if the use of the macro was appropriate here or would be well received (i.e. I think it also kinda looks ugly). It just depends if we're aiming for always using macros for save context access or not

Reply:
I do want to note though that there already seems to be three instances of using DUNGEON_KEY_COUNT with either `--` or `++`. One in parameter, and 2 in kaleido_debug